i18n/ru/package.i18n.json

```diff
@@ -0,0 +1,84 @@
+  "param.partialLightTheme": "Цвет фона для частично покрытых линий в светлой теме для покрытия ESP-IDF.",
+  "param.partialDarkTheme": "Цвет фона для частично покрытых линий в темной теме для покрытия ESP-IDF.",
+  "param.uncoveredLightTheme": "Цвет фона для непокрытых линий в светлой теме для покрытия ESP-IDF.",
+  "param.uncoveredDarkTheme": "Цвет фона для непокрытых линий в темной теме для покрытия ESP-IDF.",
+  "view.components.name": "Компоненты проекта",
+  "configuration.title": "ESP-IDF",
+  "espIdf.apptrace.archive.refresh.title": "Обновить список архива трассировки",
+  "espIdf.apptrace.archive.showReport.title": "Показать отчет о трассировке",
+  "espIdf.apptrace.customize.title": "Настроить параметры для трассировки приложения",
+  "trace.poll_period.description": "poll_period будет установлен для трассировки приложения",
+  "trace.trace_size.description": "trace_size будет установлен для трассировки приложения",
+  "trace.stop_tmo.description": "stop_tmo будет установлен для трассировки приложения",
+  "trace.wait4halt.description": "wait4halt будет установлен для трассировки приложения",
+  "trace.skip_size.description": "skip_size будет установлен для трассировки приложения",
+  "param.saveBeforeBuildDescription": "Сохраните все отредактированные файлы в рабочей области, прежде чем продолжить сборку. Хотя, проект будет собран, даже если не удается сохранить файлы",
+  "param.notificationSilentMode": "Отключите все уведомления о продлении ESP-IDF (за исключением уведомлений об ошибках)",
+  "param.saveScope": "Куда сохранять конфигурацию с помощью команд ESP-IDF с числовым значением vscode.ConfigurationTarget. Глобально = 1, Рабочая область = 2, Папка рабочей области = 3",
+  "param.rainmaker.api.server_url": "URL облачного сервера ESP-Rainmaker",
+  "esp.rainmaker.backend.sync.title": "Синхронизация с облачным сервером ESP-Rainmaker",
+  "esp.rainmaker.backend.connect.title": "Подключится к облачному серверу ESP-Rainmaker",
+  "esp.rainmaker.backend.logout.title": "Отвязать аккаунт Rainmaker",
+  "esp.rainmaker.backend.add_node.title": "Добавить новый узел",
+  "esp.rainmaker.backend.remove_node.title": "Удалить этот узел",
+  "esp.rainmaker.backend.update_node_param.title": "Обновить параметр для устройства",
+  "param.esp.rainmaker.oauth.url": "URL сервера OAuth2 для ESP-Rainmaker",
+  "idf.wssPort.description": "Порт сервера веб-сокетов для дампа ядра или заглушки GDB",
+  "esp.efuse.summary.title": "Получить резюме eFuse",
+  "espIdf.jtag_flash.title": "Прошить (с JTag)",
+  "espIdf.selectFlashMethodAndFlash.title": "Выберите Прошить метод и Прошить",
+  "idf.flashType.description": "Метод вспышки устройства, UART или JTag",
+  "espIdf.launchWSServerAndMonitor.title": "Запустите IDF мониторинг для режима CoreDump / GDB-Stub",
+  "esp_idf.initGdbCommands.description": "Одна или несколько команд xtensa-esp32-elf-gdb для настройки отладчика. Пример: \"initGdbCommands\": [ \"target remote /dev/ttyUSB0\"]",
```

Comment:
```suggestion
  "esp_idf.initGdbCommands.description": "Одна или несколько команд GDB для настройки отладчика. Пример: \"initGdbCommands\": [ \"target remote /dev/ttyUSB0\"]",
```

Reply:
I also recommend to change `xtensa-esp32-elf-gdb` -> `GDB` in English version too.